test/app/model/Pawn.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { Pawn } from '../../../src/app/model/Pawn';
+import { Square } from '../../../src/app/model/Types';
+import 'jest-extended';
+import { boardWithPieces, emptyBoard } from './BoardFixture';
+
+describe('Pawn movement', () => {
```

Comment:
Proponuję, że jakbyś napisał tutaj 'Pawn possible moves' to wtedy mozesz w kazdym z testow sie pozbyc poczatku:
`Check possible squares to go`. A wiadomo, że my to sprawdzamy :) Potem test tylko opisuje dany przypadek.